src/solvers/dgsem/basis_lobatto_legendre.jl

```diff
@@ -584,15 +587,15 @@ function gauss_lobatto_nodes_weights(n_nodes::Integer)
     N = n_nodes - 1
 
     # Calculate values at boundary
-    nodes[1] = -1.0
-    nodes[end] = 1.0
-    weights[1] = 2 / (N * (N + 1))
+    nodes[1] = -1
+    nodes[end] = 1
+    weights[1] = RealT(2) / (N * (N + 1))
     weights[end] = weights[1]
 
     # Calculate interior values
     if N > 1
-        cont1 = pi / N
-        cont2 = 3 / (8 * N * pi)
+        cont1 = RealT(pi) / N
+        cont2 = 3 / (8 * N * RealT(pi))
```

Comment:
I think we use

```suggestion
        cont1 = convert(RealT, pi) / N
        cont2 = 3 / (8 * N * convert(RealT, pi))
```

in other places, see https://trixi-framework.github.io/Trixi.jl/stable/conventions/#Non-exact-floating-point-numbers